.github/workflows/make_bundle.yml

```diff
@@ -46,7 +46,8 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install briefcase==0.3.1 tomlkit wheel dmgbuild>=1.4.2
-          python -m pip install -e .[pyside2]
+          python -m pip install PySide2==5.14.2.3
```

Comment:
Why do we need to specify `PySide2==5.14.2.3` here if we have

```python
EXTRA_REQS = ["pip", "PySide2==5.14.2.3", "scikit-image", "zarr"]
```
down below?

Reply:
I think @jni hit the nail on the head with this comment:

>  Indeed, in this comment by @DragaDoncila we see that the resource files have the Qt version embedded in the file name, while in this comment @tlambert03 mentions that he pre-builds the Qt resources — but that presumably uses the PySide2 from the build environment? In which case, unpinning Qt would have caused the Qt resource bundling to fail, and the symptoms experienced by @DragaDoncila and myself.

the key point here is that briefcase is a PEP517-compliant build system, that creates it's own little environment (using the metadata listed in `pyproject.toml`), but it bundles _whatever the current folder looks like_.  So, if we want to pre-build the icons (which is important, minimally, for OS X and gatekeeper checks), then we have to run napari once in the "global" environment _prior_ to running briefcase, and so it's important that the Qt backends (and maybe versions?) match.

Reply:
ahhh, I may understand now, thanks @tlambert03!! tricky stuff for sure